snapshot-labs/snapshot
# feat: Email notifications

* add a subscription form to email report
* add the email notification form to the profile page
* show subscription result in modal
* require wallet ownership proof
* add subscribe form to the modal after voting
* change the view of email subscription success modal

### Issues
[project](https://github.com/orgs/snapshot-labs/projects/11/views/10)

### Changes 
This PR includes email notification and subscription management functionality

### How to test
Subscription:

1. Log in to your account
2. Click on your profile in the header navigation
3. In the dropdown, select "SUBSCRIBE"
4. Fill out the email form in the modal
5. Submit
6. Sign the message
7. See success/error message

Subscription management:

1. Log in to your account
2. Click on your profile in the header navigation
3. In the dropdown, select "SUBSCRIBE"
4. Fill out the email form in the modal
5. Submit
6. Sign the message
7. Validate the email with a letter on your email provider
8. Close the form (if you missed the previous step, you need to do it and refresh the page)
9. Click on your profile in the header navigation
10. In the dropdown, select "Manage subscriptions"
11. Update your subscription preferences
12. Sign the message
13. Validate that subscription preferences are changed. It also can be checked in DB. Also, you can ask BE to send a test email.


### To-Do



### Self-review checklist
